stateresolution.go

```diff
@@ -80,43 +80,45 @@ func (r *stateResolver) addConflicted(events []Event) {
 		stateKey  string
 	}
 	offsets := map[conflictKey]int{}
-	// Split up the conflicted events into blocks with the same state key.
+	// Split up the conflicted events into blocks with the same type and state key.
+	// Separate the auth events into sepecially named lists because they have
```

Comment:
> sepecially

Other than that, this is a good explanation for why you're splitting them, thank you!